README.md

```diff
@@ -49,23 +49,30 @@ What are the types of the following expressions and what do they evaluate to?
 
 ```js
 17
-
+number 17
 1 + 2 * 3 + 4
+Number
+1 + 6 + 4
+7 + 4
+11
 
 800 / 80 / 8
```

Comment:
Don't forget to always do the whole problem. You're supposed to also list the types.